code/overview/cross-validation/k-fold.py

```diff
@@ -0,0 +1,43 @@
+# An example of K-Fold Cross Validation split
+
+import numpy
+from sklearn.model_selection import KFold
+
+# Configurable constants
+NUM_SPLITS = 3
+
+
+def print_result(kfold):
```

Comment:
The documentation is good. Output is pretty nice but the '\t' has some display issues depending on where it is used in the line. Just using a '\n' will still differentiate between the training and testing set and appear more consistent. An additional newline and label with fold number would also be helpful to differentiate between folds.